src/mumble/ChannelFilter.h

```diff
@@ -0,0 +1,21 @@
+// Copyright 2022 The Mumble Developers. All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file at the root of the
+// Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
+#ifndef MUMBLE_CHANNELFILTER_H_
+#define MUMBLE_CHANNELFILTER_H_
+
+/// Visibility modifiers used when applying the channel filter
+/// The channel the user is in will be always visible
+/// This enum is used in the client DB, only append new entries
+enum class ChannelFilter {
```

Comment:
```suggestion
enum class ChannelFilterMode {
```
That would seem like a more suitable name, doesn't it?
If this is changed, the file name and include guard should be adapted as well

Or maybe something like `ChannelFilterProperty`?